src/pages/Home.js

```diff
@@ -1,22 +1,42 @@
 import { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import TutoriesGallery from '../components/home/TutoriesGallery';
+import { useCurrentUserQuery } from '../api/usersData';
+import Login from './Login';
 import {
-  translateLeft, translateRight, updateHasReachedMaxScrolled, updateIsComputerWidth,
+  translateLeft, translateRight, updateHasReachedMaxScrolled, updateIsComputerWidth, updateUser,
 } from '../redux/tutories/tutoriesSlice';
+
 /* TODO: tutories should be fetched from API */
 
 export default function Home() {
   const dispatch = useDispatch();
+  // const [user, setUser] = useState({
+  //   loggedIn: false,
+  //   user: {},
+  // });
   // import error and status once the API is deployed
   const {
-    tutories, translated, isComputerWidth, reachedMaxScroll,
+    tutories, translated, isComputerWidth, reachedMaxScroll, user,
   } = useSelector((store) => store.tutories);
 
+  const { data: currentUser, isLoading } = useCurrentUserQuery();
+
   const amountScrollPages = Math.ceil(tutories.length / 3); // ceil always rounds up
   const itemsAmount = 3 * amountScrollPages; // total slider width / amount items that fits
   const amountToTranslate = 100 / amountScrollPages;
 
+  useEffect(() => {
+    // setUser({
+    //   loggedIn: currentUser && currentUser.logged_in,
+    //   user: (currentUser && currentUser.user) || {},
+    // });
```

Comment:
I kindly suggest you remove the commented code since it has no use case in the repository or deployed project.